generators/cucumber/templates/hooks.rb

```diff
@@ -2,6 +2,12 @@
   Capybara.use_default_driver
   Howitzer::Log.print_feature_name(scenario.feature.name)
   Howitzer::Log.print_scenario_name(scenario.name)
+  driver = Capybara.current_session.driver
```

Comment:
Also, the code must be executed once on driver initialization.
Probably, it should be like it 
https://github.com/strongqa/howitzer/blob/master/generators/config/templates/capybara.rb#L186